src/components/Navigation/SearchBar/SearchBar.vue

```diff
@@ -113,16 +114,19 @@ export default defineComponent({
       }
     };
 
-    const setActiveFilter = function(): void {
-      router.replace({
-        path: "/",
-        query: {
-          ...router.currentRoute.value.query,
-          filterPhrase: filterVal.value
-        }
-      });
-      store.commit("nav/filterFoodByCategory");
-      store.commit("nav/filterFoodList", filterVal.value);
+    const setActiveFilter = function(event: any): void {
+      if(event && event.target){
+        const filterValue = event.target.value;
+        router.replace({
+          path: "/",
```

Comment:
And then, I would change it to name route. It will avoid futute problems like route path change